ehr_billing/src/org/labkey/ehr_billing/notification/BillingNotification.java

```diff
@@ -165,13 +165,25 @@ public String getMessageBodyHTML(Container c, User u)
         endDate.add(Calendar.DATE, 1);
 
         getProjectSummaryPerCategory(u, startDate, endDate, categoryToQuery, containerMap, dataMap, totalsByCategory, fields);
+        Map<String, Map<String, Object>> additionalCategoryInfo = _notificationProvider.getAdditionalChargeCategoryInfo(u, billingContainer, startDate.getTime(), endDate.getTime());
+
+        for (String category : additionalCategoryInfo.keySet())
+        {
+            totalsByCategory.put(category, new HashMap<>());
+            Map<String, Double> totalsMap = new HashMap<>();
+            Map<String, Object> categoryInfo = additionalCategoryInfo.get(category);
+            totalsMap.put("totalCost", (Double) categoryInfo.get("totalCost"));
```

Comment:
Should any of these strings use the new methods like `getTotalCostColName()` on `InvoicedItemsProcessingService`?

Reply:
This is now refactored as part of the cleanup above